src/test/acceptance/features/overview.feature

```diff
@@ -6,12 +6,12 @@ Feature: Temporary overview page
   Scenario: The overview page is available and starts the application process
     Given I navigate to the HTBHF overview page
     When I select to start the process
-    Then I am shown the enter name page
+    Then I am shown the enter date of birth page
```

Comment:
As per the comments about keeping this more generic, you could call this `Then I am shown the first page of the application process`. I also have no objection to keeping it like this either.